components/director/internal/domain/eventing/service.go

```diff
@@ -18,6 +20,7 @@ import (
 )
 
 const (
+	shouldNormalizeLabel         = "shouldNormalize"
```

Comment:
isNormalized ?

Reply:
I don't see how it fits what we're trying to convey. By **shouldNormalize** we mean that Applications (more precisely their names) _should be normalized_ for a specific runtime. We don't say that they _are normalized_ (isNormalized) or that the runtime itself _is normalized_?

Reply:
from kyma agent PoV they will interpret shouldNormalize label as "normalization already done, skip" === isNormalized=true or atleast that was my understanding after the sync today. If the label is called shouldNormalize and has value true and the kyma agent skips normalization because we do it internally when shoulldnormalize=true, its kind of confusing (a client skips normalization when shouldNormalize is true).

Reply:
Renaming has been done.